src/AppBundle/Templating/Helper/SourceCodeHelper.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * This file is part of the Symfony package.
+ *
+ * @copyright (c) 2015, symfony-demo
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Templating\Helper;
+
+use ReflectionClass;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\Templating\Helper\Helper;
+
+/**
+ * Class SourceCodeHelper
+ *
+ * @author Rasanga Perera <rasangaperera@gmail.com>
+ */
+class SourceCodeHelper extends Helper
```

Comment:
adding a dependency on the Templating component for the Twig extension is precisely the opposite of what we are doing in Symfony 2.7+, where we are making all Twig extension usable without having the Templating component.